src/components/App/App.jsx

```diff
@@ -1,33 +1,35 @@
-import { useState } from 'react';
-import AccordionCard from '../AccordionCard/AccordionCard';
-import accordionData from '../../AccordionData/AccordionData';
-import './App.css';
+import { useState } from "react";
+import AccordionCard from "../AccordionCard/AccordionCard";
+import accordionData from "../../AccordionData/AccordionData";
+import "./App.css";
 
 function App() {
   const [accordionFacts] = useState(accordionData);
 
   return (
-    <div className="app">
+    <div className='app'>
       <AccordionList items={accordionFacts} />
     </div>
   );
 }
 
 const AccordionList = ({ items }) => (
-  <div className="accordion-wrapper">
-    <div className="accordion">
+  <div className='accordion-wrapper'>
+    <div className='accordion'>
       <TitleWrapper />
-      {items.map(({ id, title, content }) => (
-        <AccordionCard key={id} title={title} content={content} />
-      ))}
+      <div className='scroll-area'>
```

Comment:
Nice!  Great instincts with this.  Can't wait to pull from main and see how it looks!